images/egress/router/egress-router.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# OpenShift egress router setup script
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+BLANK_LINE_OR_COMMENT_REGEX="([[:space:]]*$|#.*)"
+PORT_REGEX="[[:digit:]]+"
```

Comment:
iptables-restore will fail if the data we pass it is bad, so we don't need to validate it 100% here. The regexes are about (a) distinguishing the different valid inputs (which is more important in the http proxy case (#13586) where we have to distinguish "a line which is just a hostname" from "a line which is just an IP address") and (b) making sure that when we plug them into an iptables rule, iptables will either parse it correctly or reject it entirely. (Eg, we don't want to allow `destip` to contain spaces and random iptables flags.)

But I can adjust the strictness of the checks here if people want.